modules/migration-compatibility-guidelines.adoc

```diff
@@ -10,24 +10,16 @@
 
 You must install the {mtc-full} ({mtc-short}) version that is compatible with your {product-title} version.
 
-You cannot install {mtc-short} 1.6.x on {product-title} versions 3.7 to 4.5 because the custom resource definition API versions are incompatible.
+You cannot install {mtc-short} 1.6.x on {product-title} 4.5 or earlier versions because the custom resource definition API versions are incompatible.
 
-You can migrate workloads from a source cluster with {mtc-short} 1.5.1 to a target cluster with {mtc-short} 1.6.x as long as the `MigrationController` custom resource and the {mtc-short} web console are running on the target cluster.
+You can migrate workloads from a source cluster with {mtc-short} 1.5.3 to a target cluster with {mtc-short} 1.6.x as long as the `MigrationController` custom resource and the {mtc-short} web console are running on the target cluster.
 
-[cols="1,1,2a", options="header"]
+[cols="1,1,2", options="header"]
 .{product-title} and {mtc-short} compatibility
 |===
 |{product-title} version |{mtc-short} version |{mtc-full} Operator
-|3.7 |1.5.1 |Legacy {mtc-full} Operator.
 
-Installed manually with the `operator-3.7.yml` file.
+|3.9 to 4.5 |1.5.3 |Legacy {mtc-full} Operator, installed manually with the `operator.yml` file.
```

Comment:
it should be `3.11 to 4.5`, the supported ocp version start from 3.11

Reply:
Done. Changed to "4.5 and earlier". I also saw that the procedure for upgrading the legacy operator needs to be updated, so that's done.